Sources/Purchasing/Purchases.swift

```diff
@@ -730,7 +729,7 @@ extension Purchases {
         attributionPoster.postAppleSearchAdsAttributionIfNeeded()
     }
 
-    @available(iOS 14.3, *)
+    @available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *)
```

Comment:
maybe we could do something like 
```suggestion
    # should match platform availability in https://developer.apple.com/documentation/ad_services
    @available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *)
```